src/theme/globalStyles.ts

```diff
@@ -8,7 +8,11 @@ const GlobalStyle = createGlobalStyle`
 		font-size: ${(props) => props.theme.fontSizes.regularParagraph}px;
 	}
 
-	input {
+	input,
+    textarea {
+		font-size: ${(props) => props.theme.fontSizes.regularParagraph}px;
+		font-family: ${({ theme: { fonts } }) => fonts.primaryFont};
```

Comment:
that's a weird construct (destructuring of fonts that is) :D It's valid so up to you but I'd do 
`font-family: ${({ theme }) => theme.fonts.primaryFont}` Less brackets to my eyes :D